sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/MapsSearchAsyncClient.java

```diff
@@ -95,2234 +60,923 @@
-     * List&lt;String&gt; getPolygonIds = results.getResults&#40;&#41;.stream&#40;&#41;
-     *     .filter&#40;item -&gt; item.getDataSource&#40;&#41; != null &amp;&amp; item.getDataSource&#40;&#41;.getGeometry&#40;&#41; != null&#41;
-     *     .map&#40;item -&gt; item.getDataSource&#40;&#41;.getGeometry&#40;&#41;&#41;
-     *     .collect&#40;Collectors.toList&#40;&#41;&#41;;
-     * getPolygonIds.add&#40;fuzzySearchId&#41;;
-     *
-     * if &#40;ids != null &amp;&amp; !getPolygonIds.isEmpty&#40;&#41;&#41; &#123;
-     *     System.out.println&#40;&quot;Get Polygon: &quot; + ids&#41;;
-     * &#125;
-     * </pre>
-     * <!-- end com.azure.maps.search.async.get_polygon -->
-     *
-     * @param geometryIds Comma separated list of geometry UUIDs, previously retrieved from an Online Search request.
+     * Use to get polygon data of a geographical area shape such as a city or a country region.
+     *
+     *
+     * The `Get Polygon` API is an HTTP `GET` request that supplies polygon data of a geographical area outline such as
+     * a city or a country region.
+     *
+     * @param coordinates A point on the earth specified as a longitude and latitude. Example: &amp;coordinates=lon,lat.
+     * @param view A string that represents an [ISO 3166-1 Alpha-2 region/country
+     * code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2). This will alter Geopolitical disputed borders and labels
+     * to align with the specified user region. By default, the View parameter is set to “Auto” even if you haven’t
+     * defined it in the request.
+     *
+     * Please refer to [Supported Views](https://aka.ms/AzureMapsLocalizationViews) for details and to see the available
+     * Views.
+     * @param resultType The geopolitical concept to return a boundary for. If not specified, the default is
+     * `countryRegion` result type.
+     * @param resolution Resolution determines the amount of points to send back. If not specified, the default is
+     * medium resolution.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ErrorResponseException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return this object is returned from a successful Search Polygon call.
+     * @return `GeoJSON GeocodingFeature` object that describe the boundaries of a geographical area.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<List<MapsPolygon>>> getPolygonsWithResponse(List<String> geometryIds) {
-        return withContext(context -> getPolygonsWithResponse(geometryIds, context));
+    public Mono<Boundary> getPolygons(GeoPosition coordinates, String view, BoundaryResultTypeEnum resultType, ResolutionEnum resolution) {
```

Comment:
`Boundary` isn't part of the public API, which it needs to be. In the `/swagger/readme.md` file the `custom-types` need to be updated with the models that need to be in the public API.